src/alcazar.rs

```diff
@@ -1,7 +1,5 @@
-use std::{
-    io::{BufRead, BufReader},
-    net::{IpAddr, Ipv4Addr, SocketAddr, TcpListener},
-};
+use crate::http_request::HttpRequest;
+use std::net::{IpAddr, Ipv4Addr, SocketAddr, TcpListener};
```

Comment:
There's probably a couple of unused dependencies here, what does clippy say ?

Reply:
There is no unused dependencies here :)

Reply:
Clippy had a problem on `stream.write`, call to replace by `stream.write_all`